app/src/pages/outside/loginPage/loginPage.jsx

```diff
@@ -59,28 +64,32 @@ export class LoginPage extends PureComponent {
     multipleAuth: PropTypes.string,
     showDefaultErrorNotification: PropTypes.func,
     instanceType: PropTypes.string.isRequired,
+    clearErrorAuth: PropTypes.func,
   };
   static defaultProps = {
     forgotPass: '',
     reset: '',
     errorAuth: '',
     multipleAuth: '',
     showDefaultErrorNotification: () => {},
+    clearErrorAuth: () => {},
   };
 
   componentDidMount() {
     if (this.props.errorAuth) {
       this.props.showDefaultErrorNotification({
         message: this.props.errorAuth,
       });
+      this.props.clearErrorAuth();
     }
   }
 
-  componentDidUpdate(prevProps) {
-    if (this.props.errorAuth !== prevProps.errorAuth) {
+  componentDidUpdate() {
+    if (this.props.errorAuth) {
       this.props.showDefaultErrorNotification({
         message: this.props.errorAuth,
       });
+      this.props.clearErrorAuth();
     }
```

Comment:
You can move this block into method and call it twice instead of duplicating the same logic.